Symfony/CS/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+/**
+ * Test that parses and runs the fixture *.test files found in
+ * /Fixtures/Integration.
+ *
+ * @author SpacePossum
+ */
+class IntegrationTest extends AbstractIntegrationTest
```

Comment:
@keradus AbstractIntegrationTest is meant to be part of the extension points for packages extending the tool with new fixers.
What we do in Symfony or Twig is that we put such classes in the ``Test`` namespace rather than ``Tests``: the class is part of the library API, not of the testsuite (and it is then used in the testsuite itself). This is even more important when the testsuite is in a separate folder which is not autoloadable at runtime (see Twig for instance)